server/BUILD

```diff
@@ -131,3 +132,27 @@ distribution(
     output_filename = "grakn-core-server",
     visibility = ["//visibility:public"]
 )
+
+deploy_deb(
+    name = "grakn-core-server",
+    target = ":server-binary",
+    package_dir = "/opt/grakn/core/server/",
```

Comment:
Would it make sense to call it `installation_dir`?

Reply:
@lolski it mirrors the name of [the attribute](https://github.com/bazelbuild/bazel/blob/master/tools/build_defs/pkg/pkg.bzl#L215) of `pkg_tar` (which we use in implementation)

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/4775/commits/d5853651e4308a331e4df8b2074f1df76025a8be